CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ x.x.x
 
 * Update requirements.txt to get latest version of edtf-validate
 * Updated project to python 3.7 
+* Added LevelConforms feature introduced in edtf-validate.
```

Comment:
I suggest making this a little more specific with something like "Added `levelConforms` query parameter to check `conformsLevel*` functionality now present in edtf-validate.